.github/file-paths.yaml

```diff
@@ -117,8 +117,8 @@ e2e_ci: &e2e_ci
   - ".github/workflows/e2e-*"
 
 e2e_specs: &e2e_specs
-  - "**/*.cy.*.js"
-  - "**/*.cy.*.ts"
+  - "**/*.cy.*.{js,ts}"
+  - "!e2e/test-host-app/**/*.cy.*.{js,ts}"
```

Comment:
Without excluding the host-app tests explicitly, this PR will fail the E2E tests as it thinks the host app tests need to be run there as well.

See the [failed tests here.](https://github.com/metabase/metabase/actions/runs/15974123841/job/45052569437)